src/settings/Settings.js

```diff
@@ -164,13 +164,6 @@ export const SETTINGS = {
         supportedLevels: ['account'],
         default: null,
     },
-    "feature_cross_signing": {
```

Comment:
our process requires explicit approval from product on whether or not it's okay to remove features btw. We got the all-clear for it being deprecated in this way, though to respect the process we should probably get explicit approval as though this was a real feature flag.

Also I'm somewhat less convinced that we need such an explicit checkmark these days given the product team controls when these things happen anyways. Historically it was added to prevent accidentally shipping things, which is fairly impossible now due to other checks (like human reviewers ensuring the action being taken is the right one).